### PR TITLE
cocktail sort added

### PR DESCRIPTION
**Pull request by TechnoPhasePRO**
**Fixes issue:** #4644

**Changes:**

Added code for cocktail sort in python. It sorts the array of numbers in both the directions with less complexity than the bubble sort.

### CODE REVIEW

File: code/sorting/src/cocktail_sort/cocktail_sort.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Python program for implementation of Cocktail Sort 
+# Cocktail sort traverses via both the directions alternatively
+# It works similar to bubble sort. But takes less iterations/pass.
+
+def cocktailSort(a):
```
Comment (arnavb) [inline code/sorting/src/cocktail_sort/cocktail_sort.py]: Please name your functions using `underscore_case`

File: code/sorting/src/cocktail_sort/cocktail_sort.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    n = len(a)
+    # swapped flag is set to True to enter the while loop
+    swapped = True
+    # starting index of the list
+    start = 0
+    # last index in the list
+    end = n - 1
```
Comment (arnavb) [inline code/sorting/src/cocktail_sort/cocktail_sort.py]: The comments on lines 9, and 11 are unnecessary. The comment on line 7 should explain the purpose of the variable, not what its set to.

File: code/sorting/src/cocktail_sort/cocktail_sort.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    while swapped == True:
```
Comment (arnavb) [inline code/sorting/src/cocktail_sort/cocktail_sort.py]:
```suggestion
    while swapped:
```

**Comment by arnavb** [comment]
Minus the added comments, this code is blatantly copied from [here](https://www.geeksforgeeks.org/cocktail-sort/). Please do not plagiarise other sources and write your own code yourself.

**Comment by TechnoPhasePRO** [comment]
@arnavb Sorry Sir, i will be adding a new code myself with PR .

**Pull request closed without merging**